tactics/dnet.ml

```diff
@@ -134,39 +128,6 @@ struct
 (*     | (Some s, _ | _, Some s) -> s *)
 (*     | _ -> raise Not_found *)
```

Comment:
This commented code should be removed too.

Reply:
I didn't touch it, it was there before and was not moved. It's not strictly part of the code I'm fixing either...

Reply:
It goes together with the fold_pattern stuff deleted just below

Reply:
OK, will do.

Reply:
Done.